docs/messaging-service/outgoing-api-spec.md

```diff
@@ -0,0 +1,53 @@
+# Outgoing API specification
+
+Here you can see the example request bodies that messaging service can execute when you configure it to use Custom API integrations.
+
+## SMS REST integration
+
+If you choose the Custom API integration option for your SMS gateway, the Messaging service will forward the actual message to the specified endpoint by doing
+an HTTP POST request. An example request body that will be sent, will look like:
```

Comment:
...a HTTP...

Reply:
If i do this change, the grammar check in the Intellij complains ```Use “an” instead of ‘a’ if the following word starts with a vowel sound, e.g. ‘an article’, ‘an hour’.```